nin/dasBoot/FullscreenRenderTargetPool.js

```diff
@@ -0,0 +1,39 @@
+class FullscreenRenderTargetPool {
+  constructor() {
+    this.renderTargets = [];
+    this.used = 0;
+  }
+
+  _createFullscreenRenderTarget() {
+    const renderTarget = new THREE.WebGLRenderTarget(16 * GU, 9 * GU, {
+      height: 16 * GU,
```

Comment:
Should this be 9 * GU? Also, check if it's really a valid option:
https://threejs.org/docs/#api/en/renderers/WebGLRenderTarget

Reply:
yep, meant to remove that